### ISSUE TITLE
Simple or complex partition

### ISSUE THREAD

**Issue by zhigangjiang** [zillow/zind#3]
Hi, thank you for providing this dataset. 
In your paper says:
>In total, the splits consisted of 10,305, 946, and 889 panoramas for training, validation, and testing, respectively.

and

>As a result, for the purposes of training and evaluation with HNet, we separated our layout annotations into those that had reasonable expectation of recovery from a single view point (“simple”), and those that do not (“complex”).

However, it seems that you only provided the partition(zind_partition.json) of the homes and did not provide which panorama is simple or complex. In order to be consistent with your evaluation of HNet, could you please provide a more detailed partition? Thank you.

**Comment by willhuZG** [comment]
Hi, thanks for your interest in ZInD. We will share the per-pano shape occlusion complexity labels here in the repository within the next 2-3 days. 

On recreating the HorizonNet evaluation published in the paper, the data split for that evaluation does not match the data split we have shared with the final release, for two reasons: 1) The dataset was not complete at the time of the original submission, it has since increased in size by ~2x, and 2) we removed additional panoramas/homes due to privacy concerns. As such, a new evaluation needs to be done on the final released partition. While we aim to redo this evaluation, we cannot guarantee any ETA on those results. Nonetheless, with the release of the complex/simple categorization, researchers such as yourself can correlate those with our proposed train/val/test split to create a benchmark on which they can re-train existing and new layout estimation models

**Comment by zhigangjiang** [comment]
Thank you for your work. 
As far as I know, in addition to HorizonNet providing the training code, [LED2-Net](https://github.com/fuenwang/LED2-Net)(CVPR 2021) also provides the training code. Are you interested in evaluating it?

**Comment by willhuZG** [comment]
The simple/complex labels are now available at the top level of the repository as `room_shape_simplicity_labels.json`. `True` corresponds to room shape annotations with simple occlusion, i.e. no contiguous series of occluded vertices. 

On additional evaluations, this is not currently a priority for us.
